sass/multi_sass_binary.bzl

```diff
@@ -0,0 +1,62 @@
+# Copyright 2021 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("//sass:sass.bzl", "sass_binary", "sass_library")
+
+def _basename(p):
+    """Gets the basename of a given path."""
+    return p.rpartition("/")[-1]
+
+def _strip_extension(p):
+    """Strips th Sass extension of the given path."""
+    return p[:-len(".scss")]
+
+def multi_sass_binary(name, srcs, output_style = None, sourcemap = None, **kwargs):
```

Comment:
It's unclear to me if this is actually worth the change/or if it's a good time thinking about not supporting this. The Angular use-case mentioned in the original issues seems rare & can be easily solved by a simple loop 🤷 I don't feel very strongly about either way here..